ui/tests/integration/components/info-table-row-test.js

```diff
@@ -69,8 +78,7 @@ module('Integration | Component | InfoTableRow', function (hooks) {
         @tooltipText={{this.tooltipText}}
       />`);
 
-    await triggerEvent('[data-test-value-div="test label"] .ember-basic-dropdown-trigger', 'mouseenter');
-
+    await triggerEvent('.ember-basic-dropdown-trigger', 'mouseenter');
```

Comment:
I'm guessing this is the only dropdown in this component? I don't mind the extra verbosity to explicitly select the specific dropdown in case other dropdowns are added to the component in the future but definitely a nit.

Reply:
solid call out. it's not an issue here, but that's a point to keep in mind